critiquebrainz/frontend/templates/artist.html

```diff
@@ -10,10 +10,26 @@
          {% if member.begin %}title="{{ _('Member since %(date)s', date=member.begin) }}"{% endif %}>
         {{ member.artist.name }}
       </a>
-      {% if member.end %}
-        <span class="text-muted">({{ member.begin }} - {{ member.end }})</span>
-      {% elif member.artist.disambiguation %}
-        <span class="text-muted">({{ member.artist.disambiguation }})</span>
+      {% if member.periods -%}
+        <span class="text-muted">(
+        {%- for period in member.periods -%}
+            {{ period[0] + '-' + period[1] }}
+            {%- if not loop.last -%}
+              {{ ', ' }}
+            {%- endif -%}
+        {%- endfor -%}
```

Comment:
I mean consistency is an important factor, but I think it would be like changing all fors to whiles to be consistent. They are not doing the same so I see no point in using only one version. Prove me wrong if I am not thinking clearly ;)